### PR TITLE
Independent views upon click on the 3 different panels of the app

### LINKED ISSUE #91: Feedinlib-panel does not open the popup automatically
Once the user clicked on the map, they need to click again on the popup to display the data.

### LINKED ISSUE #93: Missing Regions in Feedin-Tab
In the Feedin-Tab on the map it doesn't matter where one clicks, the same result is always returned. Regional selection should be implemented, even with data missing thus far.

### PR DESCRIPTION
**Pull request by Bachibouzouk**
This PR replaces #94 

Closes #91 and closes #93 

The NUTS records in the OEDB are for small landkreis. @christian-rli found a way to find the small NUTS from the NUTS of a larger region in #93. We could either do the OEP quey with a "look for text" in every row of the table, or use the nuts geojson @christian-rli prepared and use the `.in_()` sqlalchemy function to find records within that list.

### CODE REVIEW

**Comment by Bachibouzouk** [comment]
@jh-RLI - the functionalities work with the LOCAL_TEST is True (i.e. no Query to OEDB via OEP).
I would recommend you try to figure out the query with the nuts and you fix the `serial_views.py`

**Comment by jh-RLI** [comment]
@Bachibouzouk nuts work now for power plant register. Again im not sure why i get a pylint error with unused variable i.

**Review by jh-RLI — APPROVED** [review]
good

File: app_settings.py [hunk 1]
```diff
@@ -1,23 +1,27 @@
...
+from sqlalchemy import create_engine
+import sqlahelper
+
 
 from wam import settings  # pylint: disable=import-error
 
 # add local config file
 fred_config = ConfigObj(os.path.join(settings.BASE_DIR, 'WAM_APP_FRED', 'config', 'fred_app.cfg'))
 
+OEP_ACCESS = fred_config['WAM_APP_FRED']['OEP_ACCESS']
```
Comment (Bachibouzouk) [inline app_settings.py:13]: @christian-rli , @jh-RLI - you need to add OEP_ACCESS=OEP (or OEP_DIALECT) in the file config/fred_app.cfg

File: serial_views.py [hunk 2]
```diff
@@ -95,60 +138,80 @@ def ppr_view(request):
...
+                res_powerplant_tbl.id,
+                res_powerplant_tbl.nuts,  # added
+                res_powerplant_tbl.version,
+                res_powerplant_tbl.generation_type,
+                res_powerplant_tbl.generation_subtype,
+                res_powerplant_tbl.scenario
+            )
+            # create query
+            if OEP_ACCESS == 'OEP_DIALECT':
+                # TODO find a way to convert the column rea_geom_new to srid 4326
```
Comment (Bachibouzouk) [inline serial_views.py:161]: @christian-rli - do you know in what srid are the wkb in the table under the column rea_geom_new?

Comment (christian-rli) [reply serial_views.py:161]: QGIS tells me: srid=3035

Comment (Bachibouzouk) [reply serial_views.py:161]: Can you find a way to get the transformation between 3035 to 4326 with shapely? *we can't unfortunately make use of ST_Transform of SQLAlchemy as oedialect does not handle it yet :(

**Pull request merged**